docs/download/78.md

```diff
@@ -0,0 +1,176 @@
+---
+layout: default
+title: ICU 78
+nav_order: 898
+has_children: false
+parent: Downloading ICU
+---
+
+<!--
+© 2025 and later: Unicode, Inc. and others.
+License & terms of use: http://www.unicode.org/copyright.html
+-->
+
+# ICU 78
+
+ICU is the [premier library for software internationalization](https://icu.unicode.org/#h.i33fakvpjb7o),
+used by a [wide array of companies and organizations](https://icu.unicode.org/#h.f9qwubthqabj).
+
+## Draft page
+
+**This is an early draft of the ICU 78 download page. Nothing is published yet.**
+
+## Release Overview
+
+* Download: TODO [releases/tag/release-78-1](https://github.com/unicode-org/icu/releases/tag/release-78-1)
+* TODO [Maven: com.ibm.icu / icu4j / version 78.1](https://mvnrepository.com/artifact/com.ibm.icu/icu4j/78.1)
+
+ICU 78 updates to
+[Unicode 17](https://www.unicode.org/versions/Unicode17.0.0/)
+([blog](https://blog.unicode.org/2025/09/unicode-170-release-announcement.html)),
+including new characters and scripts, emoji, collation & IDNA changes, and corresponding APIs and implementations.
+
+It also updates to
+[CLDR 47](https://cldr.unicode.org/downloads/cldr-47)
+([alpha blog](https://blog.unicode.org/2025/09/unicode-cldr-48-alpha-available-for.html))
+locale data with new locales, and various additions and corrections.
+
+TODO: High-level summary
+
+ICU 78 and CLDR 48 are major releases, including a new version of Unicode and major locale data improvements.
+
+TODO status update:
+The Java technology preview implementation of the CLDR MessageFormat 2.0 specification has been updated to incorporate the
+[CLDR 46.1 spec](https://www.unicode.org/reports/tr35/tr35-74/tr35-messageFormat.html)
```

Comment:
LDML 48 spec

Reply:
We need to rewrite this whole thing, hence the TODO.